Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator+Download.swift

```diff
@@ -57,9 +57,13 @@ extension SwiftSDKGenerator {
       return result
     }
 
-    print("Using downloaded artifacts in these locations:")
-    for path in results.map(\.path) {
-      print(path)
+    if isVerbose {
+      logger.debug("Using downloaded artifacts in these locations:")
+      for path in results.map(\.path) {
+        logger.debug("-", metadata: ["path": .string(path.string)])
+      }
+    } else {
+      logger.info("Using downloaded artifacts from cache")
     }
```

Comment:
IMO the point of `logLevel` on `logger` is to do the filtering based on the level set, so there's no need to branch here on `isVerbose` value, assuming `logLevel` was set correctly previously.

```suggestion
    logger.info("Using downloaded artifacts from cache")
    logger.debug("Using downloaded artifacts in these locations:")
    for path in results.map(\.path) {
      logger.debug("-", metadata: ["path": .string(path.string)])
    }
```